changelog_unreleased/blog-post-intro.md

```diff
@@ -8,6 +8,6 @@ This release includes a new `singleAttributePerLine` option. This is an option t
 
 We've also added support formatting for some new JavaScript syntax proposals via Babel.
 
-If you enjoy Prettier and would like to support our work, consider sponsoring us directly via [our OpenCollective](https://opencollective.com/prettier) or by sponsoring the projects we depend on, including [typescript-eslint](https://opencollective.com/typescript-eslint), [remark](https://opencollective.com/unified), and [Babel](https://opencollective.com/babel).
+If you enjoy Prettier and would like to support our work, consider sponsoring us directly via [our OpenCollective](https://opencollective.com/prettier) or by sponsoring the projects we depend on, including [typescript-eslint](https://opencollective.com/typescript-eslint), [remark](https://opencollective.com/unified), and [Babel](https://opencollective.com/babel). Also thanks to [LINE](https://line.me/en/) and [Ubie](https://ubie.life/en) for actually sponsoring us! For more information on LINE's OSS sponsoring, please read the blog post [LINE has started to donate to Prettier – why companies should support OSS](https://engineering.linecorp.com/en/blog/support-prettier-and-oss/).
```

Comment:
```suggestion
If you enjoy Prettier and would like to support our work, consider sponsoring us directly via [our OpenCollective](https://opencollective.com/prettier) or by sponsoring the projects we depend on, including [typescript-eslint](https://opencollective.com/typescript-eslint), [remark](https://opencollective.com/unified), and [Babel](https://opencollective.com/babel). Also thanks to [LINE Corporation](https://opencollective.com/line), [Ubie](https://opencollective.com/ubie-inc) and many others for actually sponsoring us! For more information on LINE's OSS sponsoring, please read the blog post [LINE has started to donate to Prettier – why companies should support OSS](https://engineering.linecorp.com/en/blog/support-prettier-and-oss/).
```

Let's link to their opencollective account?

Reply:
I don't think we should link to their  OpenCollective. This is because the company that sponsored us would want us to see the company's website rather than the OpenCollective link.

Reply:
They have their production links on OpenCollective account.

Anyway, can you add "Airbnb" and "Nx"? They are doing generous donation monthly.

Reply:
> Anyway, can you add "Airbnb" and "Nx"? They are doing generous donation monthly.

Sure!

Btw, as you mentioned in Discord, I  think we should add sponsors in our website.